paimon-spark/paimon-spark-common/src/main/scala/org/apache/paimon/spark/execution/PaimonStrategy.scala

```diff
@@ -48,4 +59,16 @@ case class PaimonStrategy(spark: SparkSession) extends Strategy with PredicateHe
     new GenericInternalRow(values)
   }
 
+  private object PaimonCatalogAndIdentifier {
```

Comment:
Why not paimonCatalogAndIdentifier?

Reply:
It's an extractor object  with an unapply method in Scala, not a member method of class PaimonStrategy.